packages/core/src/baseResourceApi.ts

```diff
@@ -261,7 +261,11 @@ export abstract class BaseResourceAPI<ResponseType> {
     RequestParams extends object,
     T = IdEither
   >(items: T[], path: string = this.byIdsUrl, params?: RequestParams) {
-    return this.postInParallelWithAutomaticChunking({ params, path, items });
+    return this.postInParallelWithAutomaticChunking({
+      queryParams: params,
+      path: path,
+      items: items,
+    });
```

Comment:
@saagar-cognite The change looks good to me but I'm not really sure because this could also be a breaking change. Because if `callRetrieveEndpoint` actually meant to use `params` and not `queryParam` then this would break the implementation. 

I saw that you were the last one to work on this so mentioning you.

Reply:
I think the "proper" fix would be to rename the `params` prop to something more meaningful like `payload`, `body` or whatever it represents.

Reply:
I suggest to send both `params` & `queryParams` as both are supported in `postInParallelWithAutomaticChunking`.
It spread `...params` in `data` while sending to post call.

This option would be safest if this is a time intensive change.

Reply:
Also, I could not understand from ticket, what exactly is user trying to send which is not working.
Only thing I could infer is that they actually want to send `queryParams` but are not able to.

Reply:
@saagar-cognite could you ask directly in the issue? support might be able to help you get the answer from customer, i dont think they would go inside the PR to answer.

Reply:
@saagar-cognite I update the PR. Looks like the user had the same idea as us. So if they are only using the function from the core package then it should be alright